### ISSUE TITLE
Question about Instruction following

### ISSUE THREAD

**Issue by SOTAMak1r** [thu-ml/Causal-Forcing#34]
Thank you for your awesome work! While evaluating the Instruction Following metric for the causal forcing videos, I noticed that my calculated score (88, Yes rate: 94/100) is significantly higher than the value reported in the paper (56).

Here are the details of my evaluation setup:

- I used the official code from the VisionReward repository [inference-video.py](https://github.com/zai-org/VisionReward/blob/main/inference-video.py).

- I used the prompt template referenced in the paper: Does the video meet some of the requirements stated in the text "[[prompt]]"?

- The 100 test [videos](https://drive.google.com/drive/folders/19W8wgwD7x4f7wOsBDuOEl1U_gpBz6_gP) and their corresponding [prompts](https://github.com/user-attachments/files/26291952/text_align.txt) I used were the ones you provided in another [issue](https://github.com/thu-ml/Causal-Forcing/issues/25).

Could you please let me know if there are any specific configurations, hidden details, or missing steps I should be aware of when testing the Instruction Following metric?

Thanks in advance for your help!

**Comment by zhuhz22** [comment]
Hi @SOTAMak1r ,
[This](https://huggingface.co/zhuhz22/Causal-Forcing-original-paper/tree/main) is the original model under the setting used in our submitted paper: 3K videos generated by Wan2.1-1.3B, with 2K steps for AR diffusion, 1K steps for causal ODE, and 750 steps for asymmetric DMD. To pursue better performance, the open-source version changes the training steps of each stage to the recommended settings in this repo, and achieves stronger results.
For academic targets, we recommend the following:
1. Train Causal Forcing with our open-source data and recommended training steps. This setting is more advanced than the original paper version. If you are developing a new paper on a related method, you are also welcome to train your own method under this stronger setting.
2. If you want to strictly reproduce the metrics reported in our original paper, please use [this](https://huggingface.co/zhuhz22/Causal-Forcing-original-paper/tree/main) model.

Also  the 100 test [videos](https://drive.google.com/drive/folders/19W8wgwD7x4f7wOsBDuOEl1U_gpBz6_gP)  is exactly what we used in the original paper. Can you reproduce the same Instruction Following with these provided videos?
